ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss

```diff
@@ -17,11 +46,23 @@
   flex-direction: row;
   gap: $space-1;
   margin-left: $space-2;
+}
+
+.govuk-grid-column-two-thirds__info_cards_home {
```

Comment:
This can probably be it's own style and don't need to prefix with `govuk-grid-column-two-thirds` as it's now a new style and not overriding the existing styling. No need to tie this in with the govuk stuff.